src/components/App.tsx

```diff
@@ -25,11 +30,14 @@ class App extends React.Component<Record<string, unknown>, AppState> {
       isVerified: false,
       userReserves: [],
     };
+
+    // Need to bind to `this` since this function is called from the DOM, where this loses its scope.
+    this.signUserIn = this.signUserIn.bind(this);
   }
 
   private async getAccountInfo() {
-    this.web3 = await getWeb3();
-
+    // Must pass in Torus provider in order for Web3 to not use MetaMask by default.
+    this.web3 = new Web3(this.state.torus.provider);
```

Comment:
This makes sense. Wonder if it also gets passed in later as `currentProvider` when we call  `deposit(this.web3.eth.currentProvider, accounts[0]);` in L53

Reply:
yeah I think it's the similar call as I assume `this.web3.eth.currentProvider == this.state.torus.provider`